samcli/cli/main.py

```diff
@@ -58,6 +59,27 @@ def print_info(ctx, param, value):
     ctx.exit()
 
 
+def log_cmdline_info(func):
+    def wrapper(*args, **kwargs):
+        if kwargs["config_file"] and kwargs["config_env"]:
```

Comment:
Shall we test if `config_file` and `config_env` are in `kwargs` before testing their value?

We risk getting a `KeyError`.

```
if "config_file" in kwargs and "config_env" in kwargs and kwargs["config_file"] and kwargs["config_env"]
```

Reply:
Fixed this, thanks!